common.yml

```diff
@@ -1010,6 +1023,23 @@ tasks:
         test_path: "test/legacy30"
         smoke_args: "--authMechanism SCRAM-SHA-1"
 
+- name: format-go
+  commands:
+    - func: "fetch source"
+    - func: "fetch gometalinter"
+    - command: shell.exec
+      type: test
+      params:
+        working_dir: src/github.com/mongodb/mongo-tools
+        script: |
+          set -x
+          set -v
+          set -e
+          . ./set_goenv.sh
+          GOROOT="" set_goenv || exit
+          PATH=$PATH:$PWD/bin gometalinter --disable-all --enable=gofmt --vendor --exclude=src ./...
```

Comment:
This only runs gofmt, but we can extend this to run other linters later on if we think that would be helpful. The --exclude=src is because the "fetch gometalinter" task puts the "src" folder in the working directory and the gometalinter src has some gofmt errors.

Reply:
That's ironic.  We could vendor it, though, like we do with 3rf/mongo-lint.

Reply:
No longer needed - we can install the golangci-lint binary into bin/ directly without running go get.